templates/CRM/Mailing/Form/Subscribe.tpl

```diff
@@ -1,8 +1,8 @@
 {*
  +--------------------------------------------------------------------+
- | CiviCRM version 4.7                                                |
+ | CiviCRM version 4.5                                                |
```

Comment:
@triprems : Can you make sure you only make the needful changes? It seems like you were using an old version of file to make changes.